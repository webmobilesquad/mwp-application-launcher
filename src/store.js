import createStore from 'unistore';

export const store = createStore({
  applications: {
    data: null,
    loading: false,
  },
});
