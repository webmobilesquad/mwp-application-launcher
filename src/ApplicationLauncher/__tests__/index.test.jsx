import render from 'preact-render-spy';
import { Provider } from 'unistore/preact';
import { ApplicationLauncher } from '../index';
import { store } from '../../store';

describe('<ApplicationLauncher />', () => {
  test('<ApplicationLauncher /> matches snapshot', () => {
    expect(
      render(
        <Provider store={store}>
          <ApplicationLauncher />
        </Provider>,
      ),
    ).toMatchSnapshot();
  });
});
