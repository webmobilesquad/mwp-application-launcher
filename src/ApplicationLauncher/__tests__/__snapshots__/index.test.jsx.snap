// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ApplicationLauncher /> <ApplicationLauncher /> matches snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<i
  class="ApplicationLauncher"
  applicationClassName="Application"
  applicationActiveClassName="Application__Active"
>
</i>

`;
